torch/_inductor/virtualized.py

```diff
@@ -359,5 +364,13 @@ def current_node(self):
     def local_buffer_context(self):
         return _local_buffer_context._get_handler()
 
+    @property
+    def is_inference(self):
+        return _is_inference._get_handler()
+
+    @property
+    def is_backward(self):
+        return _is_backward._get_handler()
```

Comment:
nit: `V.graph.is_inference` already exists, see: https://github.com/pytorch/pytorch/blob/dcdb25453e0ddc6a83e0052fffc544d4d03cdffd/torch/_inductor/graph.py#L328. 

Could we just add `is_backward` to GraphLowering object and query that for the `is_backward` property, instead of adding this ?

Reply:
Totally, I missed the `is_inference` property. Will add `is_backward` there as well and revert this commit!

Reply:
Reverted this commit and added `is_backward` to `GraphLowering` https://github.com/pytorch/pytorch/pull/130429/commits/6a27b688e2ae57714cf34986a54e4abf6d870d9a